sim: true
carrot_goal_ratio: 0.5
max_ttc: 1.5
solver_time: 20
obey_time: true
allow_reverse: false
safety_radius: 0.1
adaptive_lookahead: false
push_configuration: true
push_limit_radius: 1.82
delivery_tolerance: 0.1
speed_lim: 0.4
deadlock_solve: true
add_noise: 0  # only works in simulation
nhttc_standalone: false  # set to false to use full system
time_error_thresh: 0.3
pure_pursuit: false # set to false to use full system